index.html

```diff
@@ -86,39 +96,98 @@ <h1 class="tit-projects">Projects</h1>
                <p>Project name goes here</p>
             </div>
             <ul class="buttons-flex">
-               <li class="li-button">html</li>
-               <li class="li-button">css</li>
+               <li class="li-button">HTML/CSS</li>
+               <li class="li-button">Ruby on Rails</li>
                <li class="li-button">javaScript</li>
             </ul>
             <button class="project-button">See this Project -&gt;</button>
          </div>            
       </article>
+      <article class="project">
+         <img src="resources/P5-IMG.png" alt="imagen projecto 1">
+         <div class="text-content">
+            <div class="pr-description">
+               <p>Project name goes here</p>
+            </div>
+            <ul class="buttons-flex">
+               <li class="li-button">HTML/CSS</li>
+               <li class="li-button">Ruby on Rails</li>
+               <li class="li-button">javaScript</li>
+            </ul>
+            <button class="project-button">See this Project -&gt;</button>
+         </div>            
+      </article>      
    </section>
-   
-   <section class="about-me">
-      <h1 class="title" id="ab-tit">About</h1>
-      <p class="parrafo"> 
-         Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
-         sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-         Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
-         nisi ut aliquip ea eiusmod
-      </p>
-      <button class="button-secc">Get my CV -&gt; </button>
-   </section>   
 
-   <section id="sec-abilities">
+   <section class="sec-works-pc">
+      <div class="s-1-1">
+         <img class="fill-cont" src="resources/project-pc-recortes/Slice-1-1-izq.png" alt="imagen">
+      </div>
+      <div class="s-1-2">
+         <article class="project">
+            <div class="text-content">
+               <div class="pr-description" id="pr-des-pc">
+                  <p>Project name goes here</p>
+               </div>
+               <ul class="buttons-flex" id="ul-btn-pc">
+               <li class="li-button">HTML/CSS/</li>
+               <li class="li-button">Ruby on Rails</li>
+               <li class="li-button">javaScript</li>
+               </ul>
+               <button class="project-button">See this Project -&gt; </button>
+            </div>            
+         </article>
+      </div>
+      <div class="s-2-2" >   
+         <img class="fill-cont" src="resources/project-pc-recortes/Slice-2-2.png" alt="imagen">
+      </div>
```

Comment:
Maybe we should add more relevant text in the alternate so persons with screen readers or a low internet connection can take advantage of that.